arelle/plugin/validate/ESEF/DTS.py

```diff
@@ -216,12 +217,12 @@ def lc3wordAdjust(word: str) -> str:
                 _("Anchoring relationships for concepts MUST use "
                   "\"http://www.esma.europa.eu/xbrl/esef/arcrole/wider-narrower\" arcrole: %(concepts)s."),
                 modelObject=extLineItemsWronglyAnchored, concepts=", ".join(sorted(str(c.qname) for c in extLineItemsWronglyAnchored)))
-        if extAbstractConcepts and val.authParam["extensionAbstractContexts"] != "allowed":
-            val.modelXbrl.log(val.authParam["extensionAbstractContexts"].upper(),
-                "ESEF.3.2.5.abstractConceptDefinitionInExtensionTaxonomy",
-                _("Extension taxonomy %(severityVerb)s define abstract concepts: concept %(concepts)s."),
-                modelObject=extAbstractConcepts, concepts=", ".join(str(c.qname) for c in extAbstractConcepts),
-                severityVerb={"warning":"SHOULD NOT","error":"MUST NOT"}[val.authParam["extensionAbstractContexts"]])
+        # if extAbstractConcepts and val.authParam["extensionAbstractContexts"] != "allowed":
+            # val.modelXbrl.log(val.authParam["extensionAbstractContexts"].upper(),
+            # "ESEF.3.2.5.abstractConceptDefinitionInExtensionTaxonomy",
+            # _("Extension taxonomy %(severityVerb)s define abstract concepts: concept %(concepts)s."),
+            # modelObject=extAbstractConcepts, concepts=", ".join(str(c.qname) for c in extAbstractConcepts),
+            # severityVerb={"warning":"SHOULD NOT","error":"MUST NOT"}[val.authParam["extensionAbstractContexts"]])
```

Comment:
Can we just remove this from the code as well, please?

Reply:
Requested change will be pushed soon.